activerecord/test/cases/attribute_methods_test.rb

```diff
@@ -32,6 +32,15 @@ def setup
     ActiveRecord::Base.send(:attribute_method_patterns).concat(@old_matchers)
   end
 
+  test "#id_value alias is defined if id column exist" do
+    new_topic_model = Class.new(ActiveRecord::Base) do
+      self.table_name = "topics"
+    end
+
+    assert_not_empty new_topic_model.attribute_names
+    assert_not_empty new_topic_model.attribute_aliases
```

Comment:
Nitpicking but if the test name explicitly wants this test to care about `id_value`, should we assert for its inclusion in `attribute_aliases`?
Otherwise current assertions just ensure that we have some columns and some default alias, which at the moment can't be anything other than `id_value` but perhaps in the future Rails can end up having more default aliases for whichever reason?

```suggestion
    assert  new_topic_model.attribute_names.include?("id")
    assert new_topic_model.attribute_aliases.key?("id_value")
  end
```